Project.toml

```
@@ -9,12 +9,19 @@ version = "2.7.0"
 Contour = "d38c429a-6771-53c6-b99e-75d170b6e991"
 Crayons = "a8cc5b0e-0ffa-5ad4-8c14-923d3ee1735f"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MarchingCubes = "299715c1-40a9-479a-aaf9-4a633d36f717"
+NaNMath = "77ba4419-2d1f-58cd-9bb1-8ffee604a2e3"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
 Contour = "0.5"
 Crayons = "4.1"
+MarchingCubes = "0.1"
+NaNMath = "0.3"
+StaticArrays = "1.3"
```

Is there any specific reason to restrict to StaticArrays 1.3? It seems that we only need type definitions `SVector` and `SMatrix`, so I believe older versions also works:

```suggestion
StaticArrays = "0.12, 1"
```

---

Good point, I'll change that.